### ISSUE TITLE
Snapchat ++ Loading issues

### ISSUE THREAD

**Issue by AlexAmpz** [eni9889/SC-PP-ISSUES#54]
I am frequently getting a "Could not refresh, please try again" message. I would get that message for about 15 mins, then everyone stories would load again and it'll work fine for a few hours or so and then I'll get the same message again... 

I have not updated to the new iOS.. currently running on iOS 10.3.3

I'm currently using Version: 10.16.1 of Snapchat ++

**Comment by uhhmaybe** [comment]
I'm getting this as well the last few days.  Only on the stories, not on the personal snaps.  None of the stories will show up, and the could not refresh error is displayed at the top.

`iPhone 7 Plus iOS 10.3.3`
`Snapchat++  v10.16.1_1.6r-167`

**Comment by AlexAmpz** [comment]
Yeah that's my exact issue.. only on stories, not personal nor snaps sent to me. Started about 2 days ago as well

**Comment by lrmtheboss** [comment]
Me too. Started about three days ago

**Comment by albino1** [comment]
When upgrading Snapchat++ via sideload I accidentally installed it as a fresh install with no metadata or anything as opposed to what usually happens where it just overwrites my previous Snapchat++.  I had to enter all of my info again, and redo my Snapchat++ settings from scratch, but when I finished I realized my Stories were working again and I was able to save them.  Before that I was getting the refresh error for over a week straight, but this seems to have fixed it, at least for now.

I thought I'd mention it in case it helps @eni9889 track down the issue.  Maybe a bad settings or database change between one of the more recent updates, who knows.

**Comment by uhhmaybe** [comment]
When upgrading Snapchat++ via sideload I accidentally installed it as a fresh install with no metadata or anything as opposed to what usually happens where it just overwrites my previous Snapchat++.  I had to enter all of my info again, and redo my Snapchat++ settings from scratch, but when I finished I realized my Stories were working again and I was able to save them.  Before that I was getting the refresh error for over a week straight, but this seems to have fixed it, at least for now.

I thought I'd mention it in case it helps @eni9889 track down the issue.  Maybe a bad settings or database change between one of the more recent updates, who knows.
